### PR TITLE
Save data

### PR DESCRIPTION
**Pull request by BouaikIlham**
# In this pull request:
 - [ ] we save all data into JSON files.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -2,13 +2,31 @@
...
 class App
   attr_accessor :people, :rentals, :books
 
   def initialize
     @rentals = []
     @books = []
     @people = []
+    @people_path = './../file/people.json'
+    @books_path = './../file/books.json'
+    @rentals_path = './../file/rentals.json'
```
Comment (zhadier) [inline app.rb]: - [ ] Please ensure that the file folder is inside the main project directory, right now the file is being created among my personal projects.

File: app.rb [hunk 2]
```diff
@@ -57,4 +75,79 @@ def list_books
...
+    File.write(@rentals_path, JSON.generate(rentals))
+  end
+
+  def load_people
+    people = JSON.parse(File.read(@people_path))
+    people.each do |person|
+      @people <<
+        if person['specialization']
+          Teacher.new(person['specialization'], person['age'], person['name'])
+        else
```
Comment (zhadier) [inline app.rb:126]: - [ ] Your recent changes seem to have introduced a bug and I can no longer create a teacher, it gives an argument error everytime I add a specialization. Kindly fix this small issue.

**Review by zhadier — CHANGES_REQUESTED** [review]
# Hi @BouaikIlham & @Kwentsir  👋 

Good job so far! 🌤️ 
There are some issues that you still need to work on to go to the next project but you are almost there! 🍓 



## Highlights
✔️ No linter errors
✔️ Github flow was set up correctly
✔️ Professional pull request title and description


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question @zhadier so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -21,6 +21,12 @@ git clone `https://github.com/Kwentsir/OOP_school_library.git`
 - [Twitter](https://twitter.com/jkwentsir)
 - [LinkedIn](https://www.linkedin.com/in/josephkwentsir/)
 
+👤 **Ilham Bouaik**
+
```
Comment (tufoinnkuo10) [inline README.md:25]: - [ optional] kindly add all the contributors info.

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Team

Your project is complete! Great job for making the requested changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Validations correctly implemented
- objects created succefully
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**